src/test/java/uk/gov/hmcts/cmc/claimstore/services/staff/content/InterestProviderTest.java

```diff
@@ -25,7 +24,7 @@
     private Interest interest;
     private InterestDate interestDate;
     private BigDecimal claimAmount;
-    private LocalDateTime submittedOn;
+    private LocalDate submittedOn;
```

Comment:
Please rename it to `issuedOn`, submitted on means slightly different thing IMO.